GRDB/Core/Configuration.swift

```diff
@@ -6,7 +6,8 @@ public struct Configuration {
     
     // MARK: - Misc options
     
-    /// If true, foreign key constraints are checked.
+    /// If true (the default), support for foreign keys is enabled.
+    /// See https://www.sqlite.org/foreignkeys.html for more information.
```

Comment:
```suggestion
    /// See <https://www.sqlite.org/foreignkeys.html> for more information.
```

(Based on https://github.com/groue/GRDB.swift/pull/1012)